src/pages/home-page.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import CityCard from "../components/CityCard";
+import {useDispatch, useSelector} from 'react-redux'
+import {selectAllCountries,changeContinent} from '../redux/countriesSlice'
+
+const HomePage = () => {
+
+  let dispatch = useDispatch()
+const countriesArray = useSelector(selectAllCountries)
+
+const [continent, setContinent] = useState("All")
+
+const handleContinentChange=(continent)=>{
+  dispatch(changeContinent(continent))
+}
+
+
+return (
+  <>
+     <button onClick={()=>handleContinentChange("All")}>All</button>
+     <button onClick={()=>handleContinentChange("Africa")}>Africa</button>
+     <button onClick={()=>handleContinentChange("Asia")}>Asia</button>
+     <button onClick={()=>handleContinentChange("Europe")}>Europe</button>
+     <button onClick={()=>handleContinentChange("Oceania")}>Ocenia</button>
+     <button onClick={()=>handleContinentChange("Americas")}>Americas</button>
+
+
+  <div className="header">
+  <h2>Air Pollution stats</h2>
+  </div>
+
+  <div className="container">
+    {
+
+   countriesArray.map((country)=>{
+  return  <CityCard   key={country.name} name={country.name} continent={country.continent} flag={country.flag} lat={country.lat} lon={country.lon}/>
+})
+    }
+  </div>
+  </>
+)
```

Comment:
- [ ] As requested by the previous reviewers, Kindly stick to the given design and fix the overflowing city cards on the home page, and It's supposed to be two columns only not 3 as you did.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/227724032-67ef9491-7064-4a73-af86-8d2b1e54b454.png)  | ![image](https://user-images.githubusercontent.com/68796232/227724027-846d6cf0-cff1-4edb-b341-5b0fa64cc837.png)  |

[Hint] I suggest using `display: grid;` like this:
```css
display: grid;
grid-template-columns: repeat(2, 1fr);
```